### PR TITLE
Add hostname to VMs ignition

### PR DESCRIPTION
**Pull request by nirarg**
Also include removal of pod network

### CODE REVIEW

File: data/data/kubevirt/bootstrap/main.tf [hunk 1]
```diff
@@ -1,11 +1,36 @@
+data "ignition_file" "hostname" {
+  mode  = "420" // 0644
```
Comment (ravidbro) [inline data/data/kubevirt/bootstrap/main.tf]: I don't understand the comment

File: data/data/kubevirt/masters/main.tf [hunk 2]
```diff
@@ -1,3 +1,27 @@
+data "ignition_file" "hostname" {
+  count = var.master_count
+  mode  = "420" // 0644
```
Comment (ravidbro) [inline data/data/kubevirt/masters/main.tf]: same

File: go.mod [hunk 3]
```diff
@@ -144,5 +144,5 @@ replace (
 	// Pin kubernetes hash
 	github.com/hashicorp/terraform-provider-kubernetes => github.com/hashicorp/terraform-provider-kubernetes v1.13.3-0.20201012211451-c24349628446
 	// Change kubevirt/terraform-provider-kubevirt to nirarg fork
-	github.com/kubevirt/terraform-provider-kubevirt => github.com/nirarg/terraform-provider-kubevirt v0.0.0-20201021155128-1c2b8ec764f0
+	github.com/kubevirt/terraform-provider-kubevirt => github.com/nirarg/terraform-provider-kubevirt v0.0.0-20201022163808-162e741630ed
```
Comment (ravidbro) [inline go.mod:147]: What is the reason for the change?
is it related to the hostname change?

Comment (nirarg) [reply go.mod:147]: Removed the pod network from the VMs

File: data/data/kubevirt/bootstrap/main.tf (hunk 1 above)
Comment (nirarg) [reply data/data/kubevirt/bootstrap/main.tf]: Copy/paste mistake, will remove

File: go.mod (hunk 3 above)
Comment (ravidbro) [reply go.mod:147]: I wouldn't bundle it to the same PR/commit

File: data/data/kubevirt/bootstrap/main.tf (hunk 1 above)
Comment (nirarg) [reply data/data/kubevirt/bootstrap/main.tf]: removed

File: data/data/kubevirt/masters/main.tf (hunk 2 above)
Comment (nirarg) [reply data/data/kubevirt/masters/main.tf]: removed

File: go.mod (hunk 3 above)
Comment (nirarg) [reply go.mod:147]: Its not the same commit, different commit for the modeules
I will squash all modules commits together, for the PR to openshift

Comment (ravidbro) [reply go.mod:147]: I don't think it should be at all in the same PR.
Here you are handling bootstrap and masters, workers are a different story.
also, for the bootstrap/masters I don't see that you removed the pods network interface

Comment (nirarg) [reply go.mod:147]: Please notice: the Module I changed was `terraform-provider-kubevirt` which not related to the workers, also you can see the pods networks removal change in the following vendor file:
https://github.com/nirarg/installer/blob/7aa3accef327f4f84ed4d4183cff7af1269b517e/vendor/github.com/kubevirt/terraform-provider-kubevirt/kubevirt/virtualmachine/convert.go#L117

Comment (ravidbro) [reply go.mod:147]: IMO, different PR
